pkg/metricsconfig/healthmetrics.go

```diff
@@ -87,15 +86,10 @@ func registerHealthMetrics(group metrics.Group) {
 	exporter.RegisterMetrics(group)
 	// cgrup rate metrics
 	cgroupratemetrics.RegisterMetrics(group)
-	// kprobe metrics
-	kprobemetrics.RegisterMetrics(group)
-	group.ExtendInitForDocs(kprobemetrics.InitMetricsForDocs)
 	// policy metrics
 	group.MustRegister(policymetrics.NewPolicyCollector())
 	// gRPC metrics
 	group.MustRegister(grpcmetrics.NewServerMetrics())
-	// missed metris
-	group.MustRegister(kprobemetrics.NewBPFCollector())
```

Comment:
I think the test on bpf-next is failing because kprobe metrics are not initialized now on linux,
I think this needs to stay, and the Init call above as well

Reply:
Sorry, I needed to call the splitter function.